### PR TITLE
Get force coefficients from WindTunnel Sim

### PR DESCRIPTION
**Pull request by IvanPombo**
Add a post-processing function to fetch the data of the force coefficients in a specific time step.

I would like to note that to compute the true force coefficients we need to a-priori know the frontal surface area. This is a step that I will add afterwards to an object class, instead of directly here.

### CODE REVIEW

File: inductiva/fluids/scenarios/wind_tunnel/post_processing.py [hunk 1]
```diff
@@ -112,6 +113,27 @@ def get_flow_plane(self,
...
+                                               "postProcessing", "forceCoeffs1",
+                                               "0", "forceCoeffs.dat")
+        force_coefficients_df = pd.read_csv(force_coefficients_path,
+                                            header=None)
+
+        # Skip the first lines of the header to read the force coefficients
+        # on the respective time_step.
+        force_coefficients.loc[0] = force_coefficients_df.loc[self.time_step +
+                                                              num_header_lines,
+                                                              0].split()
```
Comment (fabiocruz) [inline inductiva/fluids/scenarios/wind_tunnel/post_processing.py:133]: It seems to be that what you're trying to achieve here can be done with a better manipulation of `read_csv`.

- Why are you defining headers by hand? Are they written in the file? They can most likely be read from there.
- The last line is cryptic. What are you doing there?

Comment (cristianacarpinteiro) [reply inductiva/fluids/scenarios/wind_tunnel/post_processing.py:133]: I agree with Fábio. Maybe share an example of the dataframe to see if there is a better way of reading this!

Comment (IvanPombo) [reply inductiva/fluids/scenarios/wind_tunnel/post_processing.py]: I added a clarification on the docstring and process the file differently. 

Basically, I am reading a .dat file with the force coefficients over various iterations. On the file header there are several lines that are useless, so I remove them. The goal is to provide the users with the labels and the force coefficients at a respective time step.

File: inductiva/fluids/scenarios/wind_tunnel/post_processing.py [hunk 2]
```diff
@@ -112,6 +114,43 @@ def get_flow_plane(self,
...
+        with open(force_coefficients_path, "r",
+                  encoding="utf-8") as forces_file:
+            for index, line in enumerate(forces_file.readlines()):
+                # Pick the line 8 of the file:
+                # [#, Time, Cm, Cd, Cl, Cl(f), Cl(r)]
+                if index == num_header_lines:
+                    force_coefficients.append(line.split()[1:])
+                # Add the force coefficients for the time_step chosen
+                elif index == num_header_lines + self.time_step + 1:
+                    force_coefficients.append(line.split())
```
Comment (fabiocruz) [inline inductiva/fluids/scenarios/wind_tunnel/post_processing.py:145]: This is still a bit cryptic. Why are you indexing with `[1:]` the header line and not the others?

**Review by fabiocruz — COMMENTED** [review]
This is much better @IvanPombo, but I still have a question. Good job on dropping the dependency of pandas, too, btw!

File: inductiva/fluids/scenarios/wind_tunnel/post_processing.py (hunk 2 above)
Comment (IvanPombo) [reply inductiva/fluids/scenarios/wind_tunnel/post_processing.py:145]: I explain on top that is meant to drop the "#" entry on that list.

**Review by fabiocruz — APPROVED** [review]

**Pull request merged**